shared_model/interfaces/iroha_internal/transaction_batch.cpp

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "interfaces/iroha_internal/transaction_batch.hpp"
+#include "validators/field_validator.hpp"
+#include "validators/transaction_validator.hpp"
+#include "validators/transactions_collection/batch_order_validator.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    /**
+     * check if all transactions belong to the same batch
+     * @param txs transactions to be checked
+     * @return true if all transactions from the same batch and false otherwise
+     */
+    bool allTxsInSameBatch(const types::SharedTxsCollectionType &txs) {
```

Comment:
Add `static` to highlight that function symbol has not to be exposed to outer scopes of that .cpp.